components/search_engines/brave_prepopulated_engines.cc

```diff
@@ -193,6 +194,32 @@ const PrepopulatedEngine brave_yandex =
                        "part={searchTerms}&v=3&sn=5&srv=brave_desktop",
                        PREPOPULATED_ENGINE_ID_YANDEX);
 
+const PrepopulatedEngine brave_search = {
+    L"Brave Search beta",
+    L":br",
+    "https://cdn.search.brave.com/serp/favicon.ico",
```

Comment:
I assume the search team knows that we'll need to keep this file at this location forever (or maintain a redirect)?

Reply:
I originally got link from @remusao (explaining it would be a long term URL) but explicitly shared this requirement on Slack too (see https://bravesoftware.slack.com/archives/C01NNFM9XMM/p1623871694373500)